### PR TITLE
Feat/receipt

### PR DESCRIPTION
**Pull request by JoannaGrab**

### CODE REVIEW

File: everyday_fe/src/components/Receipt.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                {this.state.tags}
+                <button onClick={this.addTag}>Huhu</button>
+                {this.state.ingredients}
+                <input value={this.ingredient} onKeyDown={this.addIngredient}></input>
+            </div>
+        );
+    }
+}
+
+export default Component;
```
Comment (nabokj) [inline everyday_fe/src/components/Receipt.js]: pleas add new line at the end of a file

File: everyday_fe/src/components/Recipe.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <textarea value={description} onChange={handleDescriptionChange} />
+        <div>
+            <button onClick={handleSave}>Save</button>
+        </div>
+
+    </div>
+    )
+}
+
+export default Component;
```
Comment (nabokj) [inline everyday_fe/src/components/Recipe.js]: .

File: everyday_fe/src/components/Tag.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+function Component(props) {
+    return <div>{props.name}x</div>
+}
+
+
+export default Component;
```
Comment (nabokj) [inline everyday_fe/src/components/Tag.js]: .

File: everyday_fe/src/routes/Recipes.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+export default function Recipes() {
+    return (
+      <main style={{ padding: "1rem 0" }}>
+        <h2>Recipes</h2>
+        <p>
+          <RecipeList />
+        </p>
+      </main>
+    );
+  }
```
Comment (nabokj) [inline everyday_fe/src/routes/Recipes.js]: .

File: everyday_fe/src/routes/Savings.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export default function Savings() {
+    return (
+      <main style={{ padding: "1rem 0" }}>
+        <h2>Savings</h2>
+        <p>Here will be chart and summary.</p>
+      </main>
+    );
+  }
```
Comment (nabokj) [inline everyday_fe/src/routes/Savings.js]: .

**Review by nabokj — CHANGES_REQUESTED** [review]

File: everyday_fe/src/components/Receipt.js (hunk 1 above)
Comment (JoannaGrab) [reply everyday_fe/src/components/Receipt.js]: this file is deprecated. I'll remove it

**Review by nabokj — APPROVED** [review]
Looks good!

**Pull request merged**